dev/com.ibm.ws.org.apache.myfaces.2.3/bnd.bnd

```diff
@@ -58,7 +58,7 @@ instrument.classesExcludes: \
   org/**/*.class
 
 Include-Resource: \
-  @${repo;org.apache.myfaces.core:myfaces-impl;2.3.9;EXACT}!/META-INF/**, \
+  @${repo;org.apache.myfaces.core.impl;2.3.10;EXACT}!/META-INF/**, \
```

Comment:
Why the change from `org.apache.myfaces.core:myfaces-impl` to `org.apache.myfaces.core.impl`?

Reply:
I should revert this as it was a change by Jared: https://github.com/OpenLiberty/open-liberty/commit/a732411f4f4d89464fd4f38a47b317810b29604c#diff-8f1167cf80530cee0c71b8dd6375ad80afe10ecdb3cc197b7b0df74847b33ba9R64

Let me make an update. 
I rebased this branch a while ago, but I don't recall getting conflicts.